src/components/Quiz.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import Answers from './Answers';
+import { publish } from './Event';
+import QuestionAnswer from './QuestionAnswer';
+import Score from './Score';
+import TimerContainer from './TimerContainer';
+
+function Quiz() {
+  const [questionNumber, setQuestionNumber] = useState(1);
+  const [questions, setQuestions] = useState([]);
+  const nextQuestion = (question) => {
+    if (questionNumber < 21) {
+      setQuestions([...questions, question]);
+      publish('ResetInterval', 20);
+      setQuestionNumber(questionNumber + 1);
+    }
+  };
+  const shouldShowQuiz = questionNumber < 21;
+  return (
```

Comment:
Define two functions. renderQuiz, renderResults -> they will contain their own content and add a condition in here `shouldShowQuiz ? renderQuiz : renderResults`

Reply:
Done